src/services/textChanges.ts

```diff
@@ -347,6 +353,23 @@ namespace ts.textChanges {
             this.replaceRangeWithText(sourceFile, createRange(pos), text);
         }
 
+        public tryInsertJSDocParameters(sourceFile: SourceFile, parameters: codefix.ParameterInference[]) {

```

Comment:
Since `typeNode` is non-optional and `type` isn't used, maybe this needs its own interface.

Reply:
Good idea, not sure what to call it though.

Reply:
I went with JSDocParameter for now, although that's still a bit opaque as a name.